packages/PlanLimitsUI/src/pages/Usage/DailyResults.tsx

```diff
@@ -17,18 +17,18 @@ export default function DailyResults({ data }: Props) {
   return (
     <>
       <h2 className="text-xl mb-2">Daily usage grouped by area</h2>
-      {data.groups.map((usageGroup) => {
-        return (
-          <div key={usageGroup.name} className="my-6 border-b">
-            {!usageGroup.hideLabel ? (
-              <h2 className="text-lg mb-2">{usageGroup.name}</h2>
-            ) : (
-              <></>
-            )}
-            <div>
+      <div className="space-y-10 divide-y">
+        {data.groups.map((usageGroup) => {
+          return (
+            <div key={usageGroup.name} className="space-y-6">
```

Comment:
This looks like lot of change, but really leaning on the `space-y-6` from tailwind, rather than putting `mb-6` on things